src/components/Navigation/components/Section/Section.tsx

```diff
@@ -170,9 +172,12 @@ export function Section({
     </li>
   );
 
+  const itemsPrefixMarkup = itemsPrefix || null;
```

Comment:
This should be wrapped in an `<li>` if it exists, otherwise we'll end up outputting invalid html (e.g. `<ul><div>blah</div></ul>` is not valid as you can only have lis inside a ul

Reply:
yes that is very true. I did have an <li /> in ours but was styling that. I will add it back in for validity.